posthog/session_recordings/ai_data/ai_filter_prompts.py

```diff
@@ -0,0 +1,933 @@
+        },
+        {
+            "name": "$prev_pageview_max_content_percentage",
+            "property_type": "Numeric"
+        },
+        {
+            "name": "$prev_pageview_max_scroll",
+            "property_type": "Numeric"
+        },
+        {
+            "name": "$prev_pageview_max_scroll_percentage",
+            "property_type": "Numeric"
+        },
+        {
+            "name": "$prev_pageview_pathname",
+            "property_type": "String"
+        },
+        {
+            "name": "$raw_user_agent",
+            "property_type": "String"
+        },
+        {
+            "name": "$recording_status",
+            "property_type": "String"
+        },
+        {
+            "name": "$referrer",
+            "property_type": "String"
+        },
+        {
+            "name": "$referring_domain",
+            "property_type": "String"
+        },
+        {
+            "name": "$replay_minimum_duration",
+            "property_type": null
+        },
+        {
+            "name": "$replay_sample_rate",
+            "property_type": null
+        },
+        {
+            "name": "$replay_script_config",
+            "property_type": null
+        },
+        {
+            "name": "$screen_height",
+            "property_type": "Numeric"
+        },
+        {
+            "name": "$screen_width",
+            "property_type": "Numeric"
+        },
+        {
+            "name": "$selected_content",
+            "property_type": "String"
+        },
+        {
+            "name": "$sent_at",
+            "property_type": "String"
+        },
+        {
+            "name": "$session_id",
+            "property_type": "String"
+        },
+        {
+            "name": "$session_recording_canvas_recording",
+            "property_type": null
+        },
+        {
+            "name": "$session_recording_network_payload_capture",
+            "property_type": null
+        },
+        {
+            "name": "$session_recording_start_reason",
+            "property_type": "String"
+        },
+        {
+            "name": "$survey_id",
+            "property_type": "String"
+        },
+        {
+            "name": "$survey_iteration",
+            "property_type": null
+        },
+        {
+            "name": "$survey_iteration_start_date",
+            "property_type": null
+        },
+        {
+            "name": "$survey_name",
+            "property_type": "String"
+        },
+        {
+            "name": "$survey_questions",
+            "property_type": null
+        },
+        {
+            "name": "$survey_response",
+            "property_type": "String"
+        },
+        {
+            "name": "$time",
+            "property_type": "DateTime"
+        },
+        {
+            "name": "$timezone",
+            "property_type": "String"
+        },
+        {
+            "name": "$used_bootstrap_value",
+            "property_type": "Boolean"
+        },
+        {
```

Comment:
i think the `taxonomy.py` has this available and has properties saying what the AI assistant should ignore